File: WebServices/app/src/main/java/br/com/wolneyhqf/android/aulas/webservices/activity/MainActivity.java
package br.com.wolneyhqf.android.aulas.webservices.activity;

import android.content.Context;
import android.content.Intent;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import br.com.wolneyhqf.android.aulas.webservices.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startClubesActivity(View view){
        Intent intent = new Intent(this, ClubesActivity.class);
        startActivity(intent);
    }


}

File: 2018/Aula2411/app/src/main/java/com/eecm/aula0911/MainActivity.kt
package com.eecm.aula0911

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.view.View
import android.widget.*
import com.squareup.picasso.Picasso

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportActionBar?.title = "Formulário"
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        var imageViewDowloand = findViewById<ImageView>(R.id.imageview_download)
        Picasso.with(this)
            .load("https://upload.wikimedia.org/wikipedia/pt/7/73/Trollface.png")
            .fit()
            .into(imageViewDowloand)


    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.menu_formulario, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
        var id = item?.itemId
        if(id == android.R.id.home){
            finish()
        }else if(id == R.id.action_clear){

        }else if(id == R.id.action_close){
            finish()
        }
        return super.onOptionsItemSelected(item)
    }

}
